flight/Modules/Stabilization/stabilization.c

```diff
@@ -219,6 +228,25 @@ static void stabilizationTask(void* parameters)
 		dT = PIOS_DELAY_DiffuS(timeval) * 1.0e-6f;
 		timeval = PIOS_DELAY_GetRaw();
 		
+		// exponential moving averaging (EMA) of dT to reduce jitter; ~200points
+		// to have more or less equivalent noise reduction to a normal N point moving averaging:  alpha = 2 / (N + 1)
+		// run it only at the beginning for the first samples, to reduce CPU load, and the value should converge to a constant value
+		if (dT_filter_iteration-- > 2) {
```

Comment:
That is still a bug, but was also a bug in my brain.
Wanted to keep the variable = 1, but with eyry call the variable gets decimated.  

So needs a bugfix.

```
if (dT_filter_iteration > 1) {
			dT_filter_iteration--;
```